contributors/devel/release/issues.md

```diff
@@ -0,0 +1,185 @@
+# Targeting issues and PRs to release milestones
+
+This document describes how to target issues and PRs to a release.
+The process for shepherding issues into the release by the owner, release team, and GitHub bot
+is outlined below.
+
+## Definitions
+
+- *issue owners*: creator, assignees, and user who moved the issue into a release milestone
+- *Y days*: refers to business days (using the location local to the release-manager M-F)
+- *code slush*: starts when master branch only accepts PRs for release milestone.  no additional feature development is merged after this point.
+- *code freeze*: starts 2 weeks after code slush.  only critical bug fixes are accepted into the release codebase.
+
+## Requirements for adding an issue to the milestone
+
+**Note**: Issues with unmet label requirements will automatically be removed from the release milestone.
+
+When adding an issue to a milestone, the Kubernetes bot will check that the following
+labels are set, and comment on the issue with the appropriate instructions.  The
+bot will attempt to contact the issue creator 3 times (over 3 days)
+before automatically removing the issue from the milestone.
+
+Label categories:
+
+- SIG label owner
+- Priority
+- Issue type
+
+### SIG owner label
```

Comment:
I think for GCE/GKE, there's going to be a new sig-gcp created if the mailing list discussion pans out. test-infra would go to sig-testing?